vitrina/viisp/xml_utils.py

```diff
@@ -29,7 +29,7 @@
     "lt-personal-code",
 )
 
-user_information = ("firstName", "lastName", "email", "phoneNumber", "companyName")
+user_information = ("firstName", "lastName", "email", "phoneNumber", "companyName", "proxyType")
```

Comment:
Let's do an all caps here to indicate, that it is a constant (for users not to search for this in the function itself that is using this)

```suggestion
USER_INFORMATION = ("firstName", "lastName", "email", "phoneNumber", "companyName", "proxyType")
```